dynamic/rodcutting_test.go

```diff
@@ -0,0 +1,33 @@
+package dynamic
+
+import (
+	"testing"
+)
+
+var rodCuttingExamples = []struct {
+	price    []int
+	length   int
+	expected int
+}{
+	{[]int{0, 1, 5, 8, 9}, 4, 10},
+	{[]int{0, 2, 5, 7, 8, 0}, 5, 12},
+	{[]int{0, 1, 5, 8, 9, 10, 17, 17, 20}, 8, 22},
+	{[]int{0, 3, 5, 8, 9, 10, 17, 17, 20}, 8, 24},
+}
+
+func CutRobSolutionFunctionTemplate(t *testing.T, cutRodSolutionFunction func([]int, int) int) {
```

Comment:
```suggestion
func cutRobSolTestFunc(t *testing.T, cutRodSolFunc func([]int, int) int) {
```

Reply:
Done. Sorry, I do not know how to apply your suggestion and rename the other stuff in a single commit.